### ISSUE TITLE
MS Word only supports 9 levels of bullets

### ISSUE THREAD

**Issue by nixsee** [theohbrothers/ConvertOneNote2MarkDown#126]
## Bug
Hi guys, I'm back after a long time away from markdown! I finally tried your improvements and it is fantastic!

I've noticed one bug that is inherent to Word and pandoc... Word only supports 9 levels of bullets. If you export a OneNote page that has more than that, the resulting word doc actually does show all levels. However, the final markdown file cuts it at 9 levels and moves the remaining levels to level 1, 2, 3, etc... In fact, it completely destroys the nesting that follows it.

## Expectation

The final markdown nested levels match the levels in OneNote

## Discussion
The markdown contains some sort of divider symbol where it cuts off the 9 levels. 

This:

![image](https://user-images.githubusercontent.com/24255237/167726419-1279ac04-af84-46b0-8d28-6925772f89c3.png)

ends up like this:
![image](https://user-images.githubusercontent.com/24255237/167726661-93a3e10a-5a14-4478-9fe5-65e936a021ec.png)

Perhaps we could somehow look for any `<!-- --> `sections and remove them and indent what follows? I really don't know

**Comment by leojonathanoh** [comment]
@nixsee sorry i took this long, this is an edge case, normally notes won't be this nested. But if there's that limitations in Word docs, then there's really nothing much we can do about this. Do you really have such a nested page, or is this just a test of the limits of the conversion? Haha

**Comment by nickchomey** [comment]
I've got a ton of pages like this. I just write bullets/outlines in a stream of consciousness, one thought/bullet following from another. But, yes, it's probably an edge case and not much we can do about it...

**Comment by leojonathanoh** [comment]
it will probably be easier to split up the nested notes manually. But this is really a limitation on the part `.docx`, which should not limit the amount of nesting, even if it's not renderable on a single page.

**Comment by nickchomey** [comment]
Agreed. I already manually refactored the big notes and should just get into the habit of doing that more often. Though now that I'm on markdown, its not really a limitation. Its also easier to create nested notes with the wikilinks capabilities of most markdown tools, like Obsidian. 

Lets just close this issue...

**Comment by leojonathanoh** [comment]
I'd think one way about this converting to another format other than `.docx` that preserves all levels of bullet lists, e.g. `.html` (the only issue is `.css` styles are separated and hence formatting is lost, but the markup of `<li></li>` will still be there. Even if one uses such a format, that will mean one will have to select all pages to use `.docx` except for certain pages that will use that format, which is going to be difficult.

perhaps someone might come up with something innovative.
